# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/LayersTypes.h
# Commit: 99b2e5c55a25
# Full Hash: 99b2e5c55a25f47f9a5a0650607cd54ce966c28b
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-05 16:09:03
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add D3D11 RenderCompositor for SWGL. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95797
# ==============================================================================

diff -r 96087b511fe9 -r 99b2e5c55a25 gfx/layers/LayersTypes.h
--- a/gfx/layers/LayersTypes.h	Thu Nov 05 00:51:24 2020 +0000
+++ b/gfx/layers/LayersTypes.h	Thu Nov 05 09:14:52 2020 +0000
@@ -182,6 +182,7 @@
   DIRECT_COMPOSITION,
   CORE_ANIMATION,
   SOFTWARE,
+  D3D11,
   LAST
 };
 